plugins/base/src/main/kotlin/translators/psi/DefaultPsiToDocumentableTranslator.kt

```diff
@@ -452,7 +500,7 @@ class DefaultPsiToDocumentableTranslator(
                     type.resolve()?.let { resolved ->
                         when {
                             resolved.qualifiedName == "java.lang.Object" -> JavaObject(type.annotations())
-                            resolved is PsiTypeParameter -> getProjection(resolved)
+                            resolved is PsiTypeParameter -> getProjection(resolved).copy(extra = type.annotations())
```

Comment:
It was actually a regression from the previous PR. The annotations are calculated from `type` PsiType, not inner `resolved` PsiTypeParameter